exercises/darts/src/main/java/Darts.java

```diff
@@ -0,0 +1,14 @@
+class Darts {
+
+    Darts(double x, double y) {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
+    int score() {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
+    double distanceDartToOrigin() {
```

Comment:
You don't need to add this method since it's private 🙂 You only need to add starter implementation for any methods used in the tests. Does that make sense? 🙂

Reply:
I see. Will change :smile: